src/client/components/TimeZone/TimeZone.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import DropDown from '../Dropdown/DropDown';
+import moment from 'moment';
+import 'moment-timezone';
+
+const june = moment('2014-06-01T12:00:00Z');
+const timeZoneArr = [
+  {
+    id: 1,
+    value: june.tz('America/Los_Angeles').format(), // 5am PDT
+  },
+  {
+    id: 2,
+    value: june.tz('America/New_York').format(), // 8am EDT
+  },
+  {
+    id: 3,
+    value: june.tz('Asia/Tokyo').format(), // 9pm JST
+  },
+  {
+    id: 4,
+    value: june.tz('Australia/Sydney').format(), // 10pm EST
+  },
+];
+
+export default function TimeZoneDropDown() {
```

Comment:
Since there are no properties here, the consumer of this component cannot know what timezone was selected. You need to pass the selected value and `set` function through the props.